### PR TITLE
ownership payout contract

### PR DESCRIPTION
**Pull request by tkmct**

### CODE REVIEW

File: packages/plasma-light-client/src/LightClient.ts [hunk 1]
```diff
@@ -506,19 +508,29 @@ export default class LightClient {
...
     )
-    if (!depositContract) throw new Error('Invalid depositContractAddress')
-    await depositContract.finalizeExit(
-      exit.toProperty(this.deciderManager.getDeciderAddress('Exit')),
-      Integer.from(Number(exit.stateUpdate.range.end.raw))
+    if (!decided) {
+      throw new Error(`Exit property is not decided: ${exit}`)
+    }
+
+    await this.ownershipPayoutContract.finalizeExit(
```
Comment (syuhei176) [inline packages/plasma-light-client/src/LightClient.ts]: Ideally, I think we want to get payout contract by predicate address of StateObject.
Let's add task to Backlog.

```
const payoutContractAddress = CompiledContract(exit.stateUpdate.stateObject.predicateAddress).payoutContractAddress;
const payoutContract = PayoutContractManager.getPayoutContract(payoutContractAddress);
```

File: packages/plasma-light-client/__tests__/LightClient.test.ts [hunk 2]
```diff
@@ -321,10 +328,11 @@ describe('LightClient', () => {
...
-      expect(depositContract.finalizeExit).toHaveBeenLastCalledWith(
+      expect(mockFinalizeExit).toHaveBeenLastCalledWith(
+        exit.stateUpdate.depositContractAddress,
         exit.toProperty(client['deciderManager'].getDeciderAddress('Exit')),
-        Integer.from(Number(exit.range.end.raw))
+        exit.range.start,
+        Address.from(client.address)
       )
     })
   })
```
Comment (syuhei176) [inline packages/plasma-light-client/__tests__/LightClient.test.ts]: We should have tests catching exception "Exit property is not decided".

Comment (tkmct) [reply packages/plasma-light-client/__tests__/LightClient.test.ts]: ok right. I'll add it and merge this PR.

File: packages/plasma-light-client/src/LightClient.ts (hunk 1 above)
Comment (tkmct) [reply packages/plasma-light-client/src/LightClient.ts]: okay, but is it possible to share an interface between payout-contracts?

Comment (syuhei176) [reply packages/plasma-light-client/src/LightClient.ts]: ah, it's not same interface now. We should find the way...

Comment (tkmct) [reply packages/plasma-light-client/src/LightClient.ts]: okay. I created an issue for that.

**Pull request merged**